File: spider/data/song.py
# 歌曲实体
class Song(object):
    # 歌曲名称 歌手 专辑 歌曲url 歌曲图片url
    def __init__(self, name, artist, album, song_url, pic_url, origin, song_id, album_id):
        self.name = name
        self.artist = artist
        self.album = album
        self.song_url = song_url
        self.pic_url = pic_url
        self.origin = origin
        self.song_id = song_id
        self.album_id = album_id

    def to_dict(self):
        return {"name": self.name, "artist": self.artist, "album": self.album, "song_url": self.song_url,
                "pic_url": self.pic_url, "origin": self.origin, "song_id": self.song_id, "album_id": self.album_id}

File: player/app/src/main/java/com/example/sharemusicplayer/entity/Song.java
package com.example.sharemusicplayer.entity;

public class Song {
    private Long id;    // 歌曲id
    private String name;    // 歌曲名称
    private String artist;  // 歌手
    private String album;   // 专辑
    private String song_url;    // 歌曲链接
    private String pic_url;     // 图片链接
    private String origin;      // 来源

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public String getAlbum() {
        return album;
    }

    public void setAlbum(String album) {
        this.album = album;
    }

    public String getSong_url() {
        return song_url;
    }

    public void setSong_url(String song_url) {
        this.song_url = song_url;
    }

    public String getPic_url() {
        return pic_url;
    }

    public void setPic_url(String pic_url) {
        this.pic_url = pic_url;
    }

    public String getOrigin() {
        return origin;
    }

    public void setOrigin(String origin) {
        this.origin = origin;
    }
}
